source/assets/styles/base.css

```diff
@@ -9,12 +9,13 @@ a {
   text-decoration: none;
 }
 .backdrop {
-  background: url("../images/backgrounds/bg0.jpg");
+  background-image: url("../images/backgrounds/bg0.jpg");
```

Comment:
I know that theme.js will overwrite this with a random path that works. But this default value, `../images/backgrounds/bg0.jpg`, isn't a valid path right? I only bring this up because it will sometimes output a console error. Otherwise, everything looks cool.

Reply:
good catch